jquery.dataTables/jquery.dataTables.d.ts

```diff
@@ -629,6 +634,13 @@ declare namespace DataTables {
         data(): DataTable;
 
         /**
+        * Iterate over each selected cell, with the function context set to be the cell in question. Since: DataTables 1.10.6
+        *
+        * @param fn Function to execute for every cell selected.
+        */
+        every(fn: (cellRowIdx?: number, cellColIdx?: number, tableLoop?: number, cellLoop?: number) => void): DataTable;
```

Comment:
You're right. I've removed the optional flag from parameters, committed and pushed. Should I close this PR and open a new one that includes the commit?